ci/deploy_stf.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: "Deploy STF"
+  hosts: controller
+  tasks:
+    - name: "Set the sto_dir if it isn't already set"
+      ansible.builtin.set_fact:
+        sto_dir: '{{ ansible_env.HOME }}/{{ zuul.project.src_dir }}'
+      when: sto_dir | default('') | length == 0
+
+    - name: "Get vars common to all jobs"
+      ansible.builtin.include_vars:
+        file: "vars-zuul-common.yml"
+
+    - name: "Get scenario-specific vars"
+      ansible.builtin.include_vars:
+        file: "vars-{{ scenario }}.yml"
+
+    - name: "Log into the cluster"
+      ansible.builtin.import_role:
+        name: rhol_crc
+        tasks_from: add_crc_creds.yml
```

Comment:
In case anyone else is looking for this, it comes from here https://github.com/openstack-k8s-operators/ci-framework/blob/9f9d01c1611a9408903e2aab5f6264d406891936/ci_framework/roles/rhol_crc/tasks/add_crc_creds.yml  which is checked out by the zuul job above